cargo-shuttle/src/args.rs

```diff
@@ -183,7 +183,26 @@ pub struct InitArgs {
 
 // Helper function to parse and return the absolute path
 fn parse_path(path: &OsStr) -> Result<PathBuf, io::Error> {
-    canonicalize(path).map_err(|e| {
+    #[cfg(target_os = "windows")]
```

Comment:
@rutexd I really want to merge this. But will prefer a solution where we don't use `cfg`. Ie the same non-canonicalize code should work on both Windows and Unix. If you can push such an update, then we can merge this